continuous-testing-foundations/dotnet/Pragmatest.Wallets/test/Pragmatest.Wallets.Web.IntegrationTests/WalletControllerIntegrationTests.cs

```diff
@@ -10,98 +9,113 @@
 
-            Balance expectedBalance = new Balance() { Amount = expectedBalanceAmount };
-            BalanceResponse expectedBalanceResponse = new BalanceResponse() { Amount = expectedBalanceAmount };
+            decimal currentBalanceAmount = 4;
+            
+            // Setup Mocks
 
             Mock<IWalletService> walletServiceMock = new Mock<IWalletService>();
-
+            
+            Balance currentBalance = new Balance() { Amount = currentBalanceAmount };
             walletServiceMock
                 .Setup(walletService => walletService.GetBalanceAsync())
-                .Returns(Task.FromResult(expectedBalance));
+                .Returns(Task.FromResult(currentBalance));
 
             IWalletService walletService = walletServiceMock.Object;
 
+            // Initialize HTTP client and request data
+
             WebApplicationFactory<Startup> factory = new CustomWebApplicationFactory<Startup>(services =>
                 services.SwapTransient(provider => walletService)
             );
-
             HttpClient client = factory.CreateClient();
 
-            // Act
+            string endpoint = "Wallet/Balance";
+
+            // Set Expectations
+
+            BalanceResponse expectedBalanceResponse = new BalanceResponse() { Amount = currentBalanceAmount };
+
+            //// Act
+
             HttpResponseMessage response = await client.GetAsync(endpoint);
+            BalanceResponse actualBalanceResponse = await response.Content.ReadAsAsync<BalanceResponse>(new[] { new JsonMediaTypeFormatter() });
+
+            //// Assert
 
-            // Assert
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
 
-            BalanceResponse actualBalanceResponse = await response.Content.ReadAsAsync<BalanceResponse>(new[] { new JsonMediaTypeFormatter() });
             actualBalanceResponse.ShouldCompare(expectedBalanceResponse);
 
             walletServiceMock.Verify(walletService => walletService.GetBalanceAsync(), Times.Once);
             walletServiceMock.VerifyNoOtherCalls();
         }
 
         [Fact]
-        public async Task Deposit_Deposit10_Returns20Async()
+        public async Task Deposit_Deposit10_ReturnsOkPostDepositBalanceAsync()
         {
-            // Arrange
-            string endpoint = "Wallet/Deposit";
-
-            decimal expectedBalanceAmount = 20;
+            //// Arrange
+            
+            decimal postDepositBalanceAmount = 20;
             decimal depositAmount = 10;
 
-            Balance expectedBalance = new Balance() { Amount = expectedBalanceAmount };
-            BalanceResponse expectedBalanceResponse = new BalanceResponse() { Amount = expectedBalanceAmount };
+            // Setup Mocks 
 
             Mock<IWalletService> walletServiceMock = new Mock<IWalletService>();
 
+            Balance postDepositBalance = new Balance() { Amount = postDepositBalanceAmount };
             Deposit expectedDeposit = new Deposit { Amount = depositAmount };
```

Comment:
I think this should be changed to deposit

Reply:
Ageed. will be updated on next PR